src/rpc_server.rs

```diff
@@ -3561,23 +3561,59 @@ pub mod error {
         #[error("capacity to store exported block proposals exceeded")]
         ExportedBlockProposalStorageCapacityExceeded,
 
-        #[error(transparent)]
-        CreateTxError(#[from] tx_initiation::error::CreateTxError),
+        #[error("create transaction error: {0}")]
+        CreateTxError(String),
+
+        #[error("upgrade proof error: {0}")]
+        UpgradeProofError(String),
+
+        #[error("send error: {0}")]
+        SendError(String),
+
+        #[error("regtest error: {0}")]
+        RegTestError(String),
 
-        #[error(transparent)]
-        UpgradeProofError(#[from] tx_initiation::error::UpgradeProofError),
+        #[error("wallet error: {0}")]
+        Error(String),
```

Comment:
```suggestion
        WalletError(String),
```

Reply:
made suggested change in 1b9a52926cb7755fc853048ec76c5d066a32d47d

Reply:
Oh, I hadn't realized this was out of scope! Apologies for making the scope creep. I had thought this a simple oversight.